lago_python_client/models/payment_request.py

```diff
@@ -0,0 +1,47 @@
+from typing import List, Optional
+
+from ..base_model import BaseModel, BaseResponseModel
+from .customer import CustomerResponse
+from .invoice import InvoiceResponse
```

Comment:
this one is not needed if we add `PaymentRequestInvoiceResponse`, no?

Reply:
good catch! Thanks